app/views/admins/user_search.mobile.haml

```diff
@@ -0,0 +1,56 @@
+.col-md-9
```

Comment:
This page is also nearly the same, the only difference is `pull-right` vs `btn-block` for the submit buttons. This can be done for example with a `button_class` parameter for the partial.